bedrock/mozorg/urls.py

```diff
@@ -146,6 +146,8 @@
     page("analytics-tests/", "mozorg/analytics-tests/ga-index.html"),
     path("email-mieco/", views.mieco_email_form, name="mozorg.email_mieco"),
     page("santalocator/", "mozorg/santa-locator.html"),
+    # Builders AI Challenge - Terms and Conditions
+    page("about/legal/terms/builders-challenge", "mozorg/about/legal/builders-challenge.html"),
```

Comment:
hang on, I'm a bit confused here too - why does the url have `terms` path segment? this template should probably live in the `legal` folder: https://github.com/mozilla/bedrock/tree/main/bedrock/legal/templates/legal/terms

Reply:
Oops, I didn't realize we had that folder 👀 💦